dev/dev-env-setup-miniforge.sh

```diff
@@ -0,0 +1,211 @@
+#!/bin/bash
```

Comment:
Can we rename the script to indicate this is for M1 chip?

Reply:
renamed!